### DISCUSSION TITLE
Firefox is planning to add tab grouping!

### DISCUSSION THREAD

**Discussion by AxeldeWater** [Floorp-Projects/Floorp#1183] [Show and tell]
https://connect.mozilla.org/t5/discussions/here-s-what-we-re-working-on-in-firefox/td-p/57694

Firefox is going to add native tab grouping, as well as vertical tabs and a sidebar. This is very interesting for Floorp. Would Floorp start using Firefox's native sidebar instead of Floorp's custom-made one? And what about vertical tabs?

**Comment by CutterKnife** [comment]
They will leverage Firefox assets and optimize existing features to those new features.
This is just my guess, but I think it will be something like, the Vertical Tabs will be from Firefox, and the Floorp Workspaces will be added.
